omero/autogen_docs

```diff
@@ -14,6 +14,11 @@ WORKSPACE=${WORKSPACE:-$(pwd)}
 WORKSPACE=${WORKSPACE%/}  # Remove trailing slashes
 USER=${USER:-$(whoami)}
 
+echo "Retrieve model object"
+URL="https://latest-ci.openmicroscopy.org/jenkins/job/OMERO-server/lastSuccessfulBuild/artifact/EveryObject.rst"
```

Comment:
I am lost on which version are being updated:

- the GitHub action checks out and builds the HEAD of ome/openmicroscopy
- this step downloads an artifact generated by latest-ci (i.e. assuming HEAD)
- `omero dependencies` downloads a released server and updates the versions

Shouldn't all of this use the same version?

Reply:
I could use head. I was testing independently and could now reconcile the 2